functions-python/helpers/utils.py

```diff
@@ -126,3 +127,15 @@ def download_and_get_hash(
         if os.path.exists(file_path):
             os.remove(file_path)
         raise e
+
+
+def is_zip_file(file_path):
+    """
+    Checks if the file is a valid ZIP file.
+    :param file_path: the path to the file
+    """
+    try:
+        with zipfile.ZipFile(file_path, "r"):
```

Comment:
[question]: why not use https://docs.python.org/3/library/zipfile.html#zipfile.is_zipfile function instead?

Reply:
didn't know it existed 🤭 -- i have updated the code